galactic/distribution.yaml

```diff
@@ -1327,6 +1327,25 @@ repositories:
       url: https://github.com/vooon/ntpd_driver.git
       version: ros2
     status: maintained
+  octomap:
+    doc:
+      type: git
+      url: https://github.com/octomap/octomap.git
+      version: master
+    release:
+      packages:
+      - dynamic_edt_3d
+      - octomap
+      - octovis
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ros-gbp/octomap-release.git
+      version: 1.9.7-1
+    source:
+      type: git
+      url: https://github.com/octomap/octomap.git
+      version: devel
```

Comment:
It's somewhat odd (though allowed) to have a different branch for the source and the doc sections.  Is this what you intend?

Reply:
Thanks for spotting this - it was the suggested setting and I forgot to change it. It might have been carried over from a mistake I made on dashing or foxy.